server/repositories/TransferRepository.spec.js

```diff
@@ -74,5 +75,24 @@ describe("TransferRepository", () => {
     expect(result).lengthOf(1);
   });
 
+  it("getTokensById", async () => {
+    const data = [{
+            capture_id: "c",
+            token_id: "t",
+          }];
+    tracker.uninstall();
+    tracker.install();
+    tracker.on('query', function sendResult(query, step) {
+      [
+        function firstQuery() {
+          expect(query.sql).match(/capture_id/);
```

Comment:
Is match here a regex kind? I feel either we should do a robust match by verifying the exact query if possible.
Even if we did this test seems to be verifying just the method signature and knex but nothing significant in our code. I would prefer to skip the unit test for this Repository class if you ask me.  It would be beneficial and reduce our developer time spent on maintaining it.

Reply:
Hmm, maybe you are right, I don't realize that I added some tests maybe look not so useful, yes, it might be unnessarary, I will think it over, and another reason lead to this result is that I wrote the test first before I wrote the implementation, so when I link this model to the real system, I don't need to worry too much about that it's some problematic model, it makes integrating the code easier.